content/en/docs/architecture/component-overview.md

```diff
@@ -183,6 +183,175 @@ return
 
 From this moment on, the device is reachable to all authorized clients and devices. Resource update requests received by a particular Gateway to which the client is connected are forwarded to the [Resource Aggregate](#resource-aggregate). Successful command validation precedes storing and publishing of this event to the [Event Bus](#event-bus) to which the CoAP Gateway is subscribed. If the update request event targets the device hosted by this instance of the CoAP Gateway, an [UPDATE](https://tools.ietf.org/html/rfc7252#section-5.8.2) is forwarded over the authorized TCP channel to the device. The device response is forwarded to the [Resource Aggregate](#resource-aggregate), which issues a resource updated event, updating the resource projection and informing the client that the update was successful.
 
+#### Device Metadata
```

Comment:
This diagram is too big. Please write a {{ %note% }} under ### Device Authorization as well as under ### Resource Publish and Subscription with text:
If you want to know more about Device Twin and it's state changes, please go to: ...

Reply:
And move the diagram to device-twin.md and split it into more diagrams.